pipelines/toast_ground_sim.py

```diff
@@ -547,415 +162,173 @@ def parse_arguments(comm):
-                        moon_el1,
-                        moon_az1,
-                        moon_el2,
-                        moon_az2,
-                        moon_phase,
-                        scan,
-                        subscan,
-                    ) = line.split()
-                    start_time = start_date + " " + start_time
-                    stop_time = stop_date + " " + stop_time
-                    # Define season as a calendar year.  This can be
-                    # changed later and could even be in the schedule file.
-                    season = int(start_date.split("-")[0])
-                    try:
-                        start_time = dateutil.parser.parse(start_time + " +0000")
-                        stop_time = dateutil.parser.parse(stop_time + " +0000")
-                    except Exception:
-                        start_time = dateutil.parser.parse(start_time)
-                        stop_time = dateutil.parser.parse(stop_time)
-                    start_timestamp = start_time.timestamp()
-                    stop_timestamp = stop_time.timestamp()
-                    all_ces.append(
-                        [
-                            start_timestamp,
-                            stop_timestamp,
-                            name,
-                            float(mjdstart),
-                            int(scan),
-                            int(subscan),
-                            float(azmin),
-                            float(azmax),
-                            float(el),
-                            season,
-                            start_date,
-                        ]
-                    )
-            schedules.append([site, all_ces])
-            ftmr.stop()
-            ftmr.report_clear("Load {}".format(fn))
-
-    schedules = comm.comm_world.bcast(schedules)
-
-    tmr.stop()
-    if comm.world_rank == 0:
-        tmr.report("Loading schedule")
-    return schedules
-
-
-@function_timer
-def get_focalplane_radius(args, focalplane, rmin=1.0):
-    """ Find the furthest angular distance from the boresight
-
-    """
-    if args.focalplane_radius:
-        return args.focalplane_radius
-
-    cosangs = []
-    for det in focalplane:
-        quat = focalplane[det]["quat"]
-        vec = qa.rotate(quat, ZAXIS)
-        cosangs.append(np.dot(ZAXIS, vec))
-    mincos = np.amin(cosangs)
-    maxdist = max(np.degrees(np.arccos(mincos)), rmin)
-    return maxdist * 1.001
+    return args, comm
 
 
 @function_timer
 def load_focalplanes(args, comm, schedules):
     """ Attach a focalplane to each of the schedules.
 
+    Args:
+        schedules (list) :  List of tuples of the form
+            (`site`, `all_ces`) where `site` is a Site
+            instansce with the `telescope` field filled
+            and `all_ces` is a list of CES objects.
+    Returns:
+        detweights (dict) : Inverse variance noise weights for every
+            detector across all focal planes.
```

Comment:
are `detweights` unitless? if so let's state it

Reply:
In K_CMB^-2. I revised the docstring.